driver-testsuite/web-fixtures/advanced_form_post.php

```diff
@@ -13,11 +13,14 @@
 }
 
 $_POST['agreement'] = isset($_POST['agreement']) ? 'on' : 'off';
-ksort($_POST);
-echo str_replace('>', '', var_export($_POST, true)) . "\n";
+foreach ($_POST as $key => $value) {
+	$post_for_printing[htmlspecialchars($key, ENT_QUOTES, 'UTF-8')] = htmlspecialchars($value, ENT_QUOTES, 'UTF-8');
```

Comment:
this is broken in case the value is an array (for a multiple select for instance)